sdk/keyvault/azure.keyvault.keys.txt

```diff
@@ -0,0 +1,180 @@
+**** Package/Module azure.keyvault.keys****
+class KeyClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_key(self, name: str, **kwargs) -> bytes: ...
+	def create_ec_key(self, name: str, hsm: bool, curve: curve=None, key_operations: key_operations=None, enabled: bool=None, expires: datetime.datetime=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def create_key(self, name: str, key_type: key_type, size: int=None, key_operations: key_operations=None, enabled: bool=None, expires=None, not_before: datetime.datetime=None, tags: dict=None, curve: curve=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def create_rsa_key(self, name: str, hsm: bool, size: int=None, key_operations: key_operations=None, enabled: bool=None, expires=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def delete_key(self, name: str, **kwargs) -> ~azure.keyvault.keys.models.DeletedKey: ...
+	def get_cryptography_client(self, key: key, **kwargs) -> :class:`~azure.keyvault.keys.crypto.CryptographyClient`: ...
+	def get_deleted_key(self, name: str, **kwargs) -> ~azure.keyvault.keys.models.DeletedKey: ...
+	def get_key(self, name: str, version: str=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def import_key(self, name: str, key: key, hsm: bool=None, enabled: bool=None, not_before: datetime.datetime=None, expires=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def list_deleted_keys(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.keys.models.DeletedKey]: ...
+	def list_key_versions(self, name: str, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.keys.models.KeyProperties]: ...
+	def list_keys(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.keys.models.KeyProperties]: ...
+	def purge_deleted_key(self, name: str, **kwargs): ...
+	def recover_deleted_key(self, name: str, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def restore_key(self, backup: bytes, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
```

Comment:
In conjunction with https://github.com/Azure/azure-sdk-for-python/pull/7615/files#r333108192 what do folks think of `restore_key_backup` or `restore_backup`?